### ISSUE TITLE
Select list: select multiple

### ISSUE THREAD

**Issue by whittaker007** [benzler/react-spreadsheet-import#4]
It would be very useful if there was an option in the select widget to allow multiple values. I guess you'd also need a setting for the divider to split on which would probably default to a comma.

**Comment by benzler** [comment]
@whittaker007 I agree, this would definitely be a cool additional feature. I don’t have the capacity to implement it right now, but feel free to submit a PR if you're interested.
